e2e/cypress/support/commands.ts

```diff
@@ -163,10 +163,81 @@ Cypress.Commands.add('cleanupV2IAMObjectsByIDPrefixes',
   (idPrefix: string, iamObjects: string[]) => {
 
   iamObjects.forEach((iamObject) => {
-    cleanupV2IAMObjectByIDPrefix(idPrefix, iamObject);
+    if (iamObject === 'projects') {
+      cleanupProjectsByIDPrefixes(idPrefix);
```

Comment:
We now need a special case for deleting projects, because of the new rules added in this PR.